src/win.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2005-2019 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
```

Comment:
Please add a file header describing the intent of this file.

Reply:
Sure.